apps/web/pages/team/[slug].tsx

```diff
@@ -99,22 +99,22 @@ function TeamPage({
                 });
               }}
               data-testid="event-type-link"
-              className="flex justify-between">
-              <div className="flex-shrink">
-                <div className="flex flex-wrap items-center space-x-2 rtl:space-x-reverse">
-                  <h2 className=" text-default text-sm font-semibold">{type.title}</h2>
-                </div>
-                <EventTypeDescription className="text-sm" eventType={type} />
+              className="absolute inset-0"
+            />
```

Comment:
What is the purpose of having a self closing tag for Link?

For me it makes more sense to have the elements directly inside it.